static/app/views/performance/transactionSummary/transactionEvents/operationSort.tsx

```diff
@@ -0,0 +1,285 @@
+import {Component, ComponentClass, ReactPortal} from 'react';
+import ReactDOM from 'react-dom';
+import {Manager, Popper, Reference} from 'react-popper';
+import {browserHistory} from 'react-router';
+import styled from '@emotion/styled';
+import {Location, LocationDescriptorObject} from 'history';
+import * as PopperJS from 'popper.js';
+
+import {GetActorPropsFn} from 'app/components/dropdownMenu';
+import MenuItem from 'app/components/menuItem';
+import Radio from 'app/components/radio';
+import {t} from 'app/locale';
+import {TableData} from 'app/utils/discover/discoverQuery';
+import EventView from 'app/utils/discover/eventView';
+
+export type TitleProps = Partial<ReturnType<GetActorPropsFn>>;
+
+type Props = {
+  title: ComponentClass<TitleProps>;
+  eventView: EventView;
+  tableMeta: TableData['meta'];
+  location: Location;
+};
+
+type State = {
+  isOpen: boolean;
+};
+
+class OperationSort extends Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+
+    let portal = document.getElementById('transaction-events-portal');
```

Comment:
Not a blocker for this PR but I'm starting to think we should have a component that handles the popper JS pieces since there's a decent amount of copy-paste going on.